app/assets/javascripts/utilities/timeAgo.js

```diff
@@ -25,15 +25,23 @@ function secondsToHumanUnitAgo(seconds) {
   return wholeUnits + ' ' + unitName + ' ago';
 }
 
-function timeAgo(oldTimeInSeconds, maxDisplayedAge = 60 * 60 * 24 - 1) {
+function timeAgo({
```

Comment:
Made this one parameter as an object. This simplifies things for using this in Preact and non-Preact code. The reason being is we do not want to [dangerously set innerHTML](https://zhenyong.github.io/react/tips/dangerously-set-inner-html.html), hence the `formatter` property on the parameter.

Reply:
Although I need this function in the new `Article />` components, I'm going to pull it out of this PR to reduce the size of this one.